README.md

```diff
@@ -47,23 +47,17 @@
 
 <img src="https://raw.githubusercontent.com/PennyLaneAI/pennylane/master/doc/_static/code.png" width="400px" align="right">
 
-- *Machine learning on quantum hardware*. Connect to quantum hardware using **PyTorch**, **TensorFlow**, **JAX**, **Keras**, or **NumPy**. Build rich and flexible hybrid quantum-classical models.
+- <strong>*Program quantum computers*</strong>. Build quantum circuits with a wide range of state preparations, gates, and measurements. Run on [high-performance simulators](https://pennylane.ai/performance/) or various hardware devices, with advanced features like mid-circuit measurements and error mitigation.

```

Comment:
'various hardware devices' could have a link to the devices page

Reply:
Agreed! Just updated this, https://github.com/PennyLaneAI/pennylane/pull/6411/commits/b999c3a9430703c5ce1994e425ee264ece96e728